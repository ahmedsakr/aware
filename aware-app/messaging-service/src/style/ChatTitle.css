#chat-title {
    margin: 0px;
    display: block;
    text-align: left;
    color: white;
    font-size: 24px;
    font-weight: bold;
    margin-top: 10px;
    background-color: #a2abab;
}

#chat-title > p {
    margin: 0px;
    padding-left: 5px;
}