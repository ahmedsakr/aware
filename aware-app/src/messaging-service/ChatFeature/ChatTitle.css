#chat-title {
    margin: 0px;
    display: block;
    text-align: left;
    color: white;
    font-size: 24px;
    font-weight: bold;
    background-color: #a2abab;
    height: 5%;
}

#chat-title > p {
    margin: 0px;
    padding-left: 5px;
}