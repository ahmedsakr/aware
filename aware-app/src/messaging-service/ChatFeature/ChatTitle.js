import React, { Component } from 'react';
import './ChatTitle.css'

class ChatTitle extends Component {

    render() {
        const {chatTitle} = this.props;

        return(
            <div id="chat-title">
                <p>{chatTitle} - Course Discussion</p>
            </div>
        );
    }
}

export default ChatTitle;
