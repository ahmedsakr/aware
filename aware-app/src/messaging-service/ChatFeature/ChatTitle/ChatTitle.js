  import React, { Component } from 'react';
import './ChatTitle.css'

class ChatTitle extends Component {

    render() {
        const {course} = this.props;

        return(
            <div id="chat-title">
                <p>{course} - Course Discussion</p>
            </div>
        );
    }
}

export default ChatTitle;
