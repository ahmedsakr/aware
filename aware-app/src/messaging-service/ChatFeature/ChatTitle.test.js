import React from 'react';
import { shallow } from 'enzyme';

import ChatTitle from './ChatTitle';

describe('ChatTitle', () => {
  it('should render correctly in "debug" mode', () => {

    const component = shallow(<ChatTitle debug />);
  
    expect(component).toMatchSnapshot();
  });
});