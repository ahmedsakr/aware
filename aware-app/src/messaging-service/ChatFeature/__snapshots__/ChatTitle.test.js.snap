// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChatTitle should render correctly in "debug" mode 1`] = `
<div
  id="chat-title"
>
  <p>
     - Course Discussion
  </p>
</div>
`;
